app/views/content_item_signups/new.html.erb

```diff
@@ -22,32 +21,27 @@
                    track_category: "email_subscriptions",
                  }) do %>
 
+      <%
+         radio_items = @subscription.child_taxons.each.map do |taxon|
```

Comment:
Does this need `each.map` - I'm wondering if it's a mistake or a fruity data structure?

Reply:
Haha, no - not sure how that got in there. Removed 👍.

Reply:
It looks like it was in the previous version, so more a c+p issue than you going mad 😁